modules/update-service-mirror-release.adoc

```diff
@@ -21,12 +21,7 @@ endif::[]
 ifdef::openshift-origin[]
 * You have created a pull secret for your mirror repository.
 endif::[]
-* If you use self-signed certificates that do not set a Subject Alternative Name, you must precede the `oc` commands in this procedure with `GODEBUG=x509ignoreCN=0`. If you do not set this variable, the `oc` commands will fail with the following error:
-+
-[source,terminal]
-----
-x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0
-----
+* If you use self-signed certificates, you have specified a Subject Alternative Name in the certificates.
```

Comment:
Consider "You may use self-signed certificates if you have specified a Subject Alternative Name for the certificate."

Reply:
To be consistent with the CommonName workaround PR mentioned in the description I will leave as is. But will bring up rewriting when I tackle merging the the two files as discussed [here](https://coreos.slack.com/archives/C85JYPHL3/p1644927941371739).